stubs/pyvmomi/pyVmomi/vmodl/__init__.pyi

```diff
@@ -1,6 +1,25 @@
 from typing import Any
 
+from fault import *
+from query import *
```

Comment:
Is this the correct way to re-export them? Currently I'm getting
![image](https://user-images.githubusercontent.com/5418540/160637957-c1f00091-573d-4af2-a0c8-3426c3665ca6.png)

Reply:
Actually i do not want to re-export them, so this import was unnecessary, but my IDE (IntelliJ 2021.3.3 with Python plugin 213.7172.26) still complains

Reply:
How do you import `vmodl`? You could try `from . import query as query` in the stubs.

Reply:
In my code: `from pyVmomi import vmodl`